assets/js/view/game.rb

```diff
@@ -151,15 +156,19 @@ def route_anchor
       @app_route.split('#')[1]
     end
 
+    def round_info
+      name = @game.round.class.name.split(':').last
+      num = @game.round.operating? ? "#{@game.turn}.#{@game.round.round_num}" : @game.turn
+      [name, num]
+    end
+
     def render_round
-      name = @round.class.name.split(':').last
-      description = @round.operating? ? "#{@game.turn}.#{@round.round_num}" : @game.turn
-      description = "#{description} - #{@round.description}"
-      h(:div, { style: { 'font-weight': 'bold' } }, "#{name} Round #{description}")
+      name, num = round_info
+      h(:div, { style: { 'font-weight': 'bold' } }, "#{name} Round #{num} - #{@game.round.description}")
     end
 
     def render_action
-      case @round
+      case @game.round
```

Comment:
why is this changed?

Reply:
@round was only set in render_game, but new code requires it elsewhere, and I didn't want to leave half the code referring to @round and half to @game.round when they are the same. The alternative is to set @round earlier/higher so it's available everywhere it's needed? (replying to this for posterity; likely this will be addressed when addressing your wider scope requirements later in the PR)